lighthouse-core/lib/minify-trace.js

```diff
@@ -37,6 +37,8 @@ const traceEventsToAlwaysKeep = new Set([
   'ResourceFinish',
   'ResourceReceivedData',
   'EventDispatch',
+  'Animation',
+  'CompositeAnimation',
```

Comment:
Is this worth it's own PR?  In hindsight I think it doesn't fit here.

Reply:
nah just these two lines to add are good here 👍 

I think they belong in `traceEventsToKeepInProcess` though